docs/reference_guide.md

```diff
@@ -168,15 +168,21 @@ OPTIONS:
     --version      bpftrace version
 
 ENVIRONMENT:
-    BPFTRACE_STRLEN             [default: 64] bytes on BPF stack per str()
-    BPFTRACE_NO_CPP_DEMANGLE    [default: 0] disable C++ symbol demangling
-    BPFTRACE_MAP_KEYS_MAX       [default: 4096] max keys in a map
-    BPFTRACE_MAX_PROBES         [default: 512] max number of probes bpftrace can attach to
-    BPFTRACE_MAX_BPF_PROGS      [default: 512] max number of generated BPF programs
-    BPFTRACE_CACHE_USER_SYMBOLS [default: auto] enable user symbol cache
-    BPFTRACE_VMLINUX            [default: none] vmlinux path used for kernel symbol resolution
-    BPFTRACE_BTF                [default: none] BTF file
-    BPFTRACE_STACK_MODE         [default: bpftrace] Output format for ustack and kstack builtins
+    BPFTRACE_BTF                        [default: none] BTF file
+    BPFTRACE_CACHE_USER_SYMBOLS         [default: (see below)] enable user symbol cache
+    BPFTRACE_CAT_BYTES_MAX              [default: 10k] maximum bytes read by cat builtin
+    BPFTRACE_DEBUG_OUTPUT               [default: 0] outputs bpftrace's runtime debug messages to the trace_pipe
+    BPFTRACE_LOG_SIZE                   [default: 1000000] log size in bytes
+    BPFTRACE_MAX_BPF_PROGS              [default: 512] max number of generated BPF programs
+    BPFTRACE_MAP_KEYS_MAX               [default: 4096] max keys in a map
+    BPFTRACE_MAX_PROBES                 [default: 512] max number of probes bpftrace can attach to
+    BPFTRACE_MAX_TYPE_RES_ITERATIONS    [default: 0] number of levels of nested field accesses for tracepoint args
+    BPFTRACE_NO_CPP_DEMANGLE            [default: 0] disable C++ symbol demangling
+    BPFTRACE_PERF_RB_PAGES              [default: 64] number of pages to allocate per CPU for perf ring buffer
+    BPFTRACE_STACK_MODE                 [default: bpftrace] Output format for ustack and kstack builtins
+    BPFTRACE_STRLEN                     [default: 64] bytes on BPF stack per str()
+    BPFTRACE_STR_TRUNC_TRAILER          [default: `..`] trailer to add to strings that were truncated
+    BPFTRACE_VMLINUX                    [default: none] vmlinux path used for kernel symbol resolution
```

Comment:
Any reason to only alphabetize in the copy-paste in docs and not the binary too? I have a minor preference to not modify the env vars in the binary so git-blame works better but would also like consistency

Reply:
I think I just forgot to alphabetize in the binary. Personally, I think it's worth the consistency but don't feel strongly.

Reply:
Yeah I guess we shouldn't make ux decisions based on git blame. There are other ways to blame an env var addition anyways. Feel free to alphabetize the whole thing